build/distributions/CentOS/7/x86_64/DVD/ks/dvd/auto.cfg

```diff
@@ -285,24 +317,22 @@ cat << EOF >> /etc/init.d/user-data
 #!/bin/sh
 udlocation="/var/simp/user-data"
 udaddress="169.254.169.254"
-
 case \$1 in
    start)
       echo "Retrieving user-data"
-      mkdir -p '/var/simp'
-      curl -k "https://\$udaddress/latest/user-data" > \$udlocation
+      curl "https://\$udaddress/latest/user-data" > \$udlocation
```

Comment:
Why was the -k option removed?  Are we guaranteed there are valid certs for this operation?